store/copr/coprocessor.go

```diff
@@ -1042,6 +1083,9 @@ func (worker *copIteratorWorker) buildCopTasksFromRemain(bo *Backoffer, lastRang
 	if worker.req.Streaming && lastRange != nil {
 		remainedRanges = worker.calculateRemain(task.ranges, lastRange, worker.req.Desc)
 	}
+	if worker.req.Paging && lastRange != nil {
```

Comment:
`Paging` and `Streaming` should not be turned on at the same time, we need add some ensurance logic.